invenio_users_resources/records/api.py

```diff
@@ -428,9 +428,9 @@ class DomainAggregate(BaseAggregate):
 
     @classmethod
     def get_record(cls, id_):
-        """Get the user via the specified ID."""
+        """Get the domain via the specified ID."""
         with db.session.no_autoflush:
-            domain = current_datastore.find_domain(id_)
+            domain = current_datastore.find_domain_by_id(id_)
```

Comment:
```suggestion
            domain = current_datastore.find_domain(id_)
```

minor: see https://github.com/inveniosoftware/invenio-accounts/pull/500#discussion_r1831443373, which would mean no need to change anything here.